docs/admins.rst

```diff
@@ -345,3 +345,108 @@ containing "enable_v1 = true".
 Pushing built images to Pulp
 ----------------------------
 
+Priority of Container Image Builds
+----------------------------------
+
+For a build system it's desirable to prioritize different kinds of builds in
+order to better utilize resources. Unfortunately, OpenShift's scheduling
+algorithm does not support setting a priority value for a given build. To
+achieve some sort of build prioritization, we can leverage node selectors to
+allocate different resources to different build types.
+
+Consider the following types of container builds builds:
+
+- *scratch build*
+- *explicit build*
+- *auto rebuild*
+
+As the name implies, *scratch builds* are meant to be used as a one-off
+unofficial container build. No guarantees are made for storing the created
+container images long term. It’s also not meant to be shipped to customers.
+These are clearly low priority builds.
+
+*Explicit builds* are those triggered by a user, either directly via fedpkg/koji
+CLI, or indirectly via pungi (as in the case of base images). These are official
+builds that will go through the normal life cycle of being tested and,
+eventually, shipped.
+
+*Auto rebuilds* are created by OpenShift when a change in the parent image is
+detected. It’s likely that layered images should be rebuilt in order to pick up
+changes in latest parent image.
+
+For any *explicit build* or *auto rebuild*, they may or may not be high
+priority. In some cases, a build is high priority due to a security fix, for
+instance. In other cases, it could be due to an in-progress feature. For this
+reason, it cannot be said that all *explicit builds* are higher priority than
+*auto rebuilds*, or vice-versa.
+
+However, *auto rebuilds* have the potential of completely consuming OSBS’s
+infrastructure. There must be some mechanism to throttle the amount of *auto
+rebuilds*. For this reason, OSBS uses a different node selector for each
+different build type:
+
+- *scratch build*: builds_scratch=true
```

Comment:
This is actually configured in osbs-client secret. It can be named whatever the user wants. It's beyond the scope of this change though.

Reply:
Are these at least the defaults though?

Reply:
Actually there are no defaults. If no node selector is specified for a given type, no node selector is used when submitting builds of that type.